policyengine_us/variables/gov/states/mt/tax/income/mt_regular_income_tax_indiv.py

```diff
@@ -0,0 +1,36 @@
+from policyengine_us.model_api import *
+
+
+class mt_regular_income_tax_indiv(Variable):
+    value_type = float
+    entity = Person
+    label = "Montana income (subtracting capital gains before 2024) tax before refundable credits, when married couples file separately"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.MT
+
+    def formula(person, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.main
+        taxable_income = person("mt_taxable_income_indiv", period)
+        filing_status = person.tax_unit(
+            "state_filing_status_if_married_filing_separately_on_same_return",
+            period,
+        )
+        status = filing_status.possible_values
+        if p.capital_gains.in_effect:
+            capital_gains = person("long_term_capital_gains", period)
+            taxable_income = max_(taxable_income - capital_gains, 0)
```

Comment:
does MT define taxable income as this quantity from 2024 on? if so should we move this logic to a taxable income variable?

Reply:
I didn't see specific instructions on excluding the capital gains from the taxable income, although the mt_capital_gain_credit is actually removed from the non_refundable credits. The instructions just give the new calculation on the capital gains income like below: 
![image](https://github.com/PolicyEngine/policyengine-us/assets/97819588/b1c02d87-8f32-4505-b42f-528fef2101df)

Reply:
>I didn't see specific instructions on excluding the capital gains from the taxable income

that's what this line of code does though

Reply:
Got it. I'll make this change to the taxable income variable.

Reply:
For clarity, the thought process was that if capital gains are taxed at a separate rate from other taxable income that the logic will be to reduce the taxable income by capital gains before applying rates

Reply:
But we should also create a mt_taxable_income.py

Reply:
Do we need to just reduce the capital gains from the `mt_taxable_income_indiv` and `mt_taxable_income_joint`? And if we create a `mt_taxable_income.py`, how can we separate the individual and joint situation?